static/app/components/replays/breadcrumbs/replayTimelineSpans.tsx

```diff
@@ -78,7 +87,8 @@ const Spans = styled('ul')`
   pointer-events: none;
 `;
 
-const Span = styled('li')<{startPct: number; widthPct: number}>`
+const Span = styled('li')<{activeTab: TabKey; startPct: number; widthPct: number}>`
+  ${p => p.activeTab !== 'network' && 'cursor: pointer;'}
```

Comment:
It will still be clickable when the cursor is `pointer` or not pointer, maybe we want a different indication that clicking won't really change much (or when you click we could put some strong focus on the network tab itself to draw the viewer in)